### DISCUSSION TITLE
not support math in md-files for sphinx？

### DISCUSSION THREAD

**Discussion by berlin2123** [executablebooks/MyST-Parser#882] [Ideas]
I am using markdown files to write sphinx doc, with using myst_parser extension.
all works fine, except maths.
```
$x=3y$ or $$y=4x$$ should be math, as markdown, however not works.
```
the maths inside rst file, works fine. Thus, mathjax is works fine.

**Comment by berlin2123** [comment]
So, MyST doesn't support converting maths from markdown files ?

**Reply by chrisjsewell** [reply]
Heya, you have to enable it: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#dollar-delimited-math

```python
myst_enable_extensions = ["dollarmath"]
```
